src/build.gradle

```diff
@@ -66,18 +66,27 @@ task startDatabase(dependsOn: ':loadPropertiesTask') {
         String portBinding = rootProject.ext.props.get('testdb_external_port') + ':' +  rootProject.ext.props.get('testdb_exposed_port')
         String dbVersion = file("config/db_version").text
         String image = rootProject.ext.props.get('testdb_image') + ":" + dbVersion
+        String container = rootProject.ext.props.get('testdb_container')
         println "Database image: " + image
         exec {
             commandLine 'docker', 'pull', image
         }
         exec {
-            commandLine 'docker', 'run', '-d', '--rm', '-p', portBinding, '--name', 'montagu-api-test_database', image
+            commandLine 'docker', 'run',
+                    '-d',
+                    '--rm',
+                    '-p', portBinding,
+                    '--name', container,
+                    "--entrypoint", "start-with-config.sh",
+                    image,
+                    "/postgresql.test.conf"
```

Comment:
You can see the script it is running here: https://github.com/vimc/montagu-db/blob/master/start-with-config.sh

It just runs the normal entrypoint for the container, but first it concatenates the given conf file (in this case `/postgresql.test.conf`) onto the end of the main postgresql.conf file.